hw/ip/otbn/dv/verilator/otbn_top_sim_waivers.vlt

```diff
@@ -6,3 +6,5 @@
 
 lint_off -rule WIDTH -file "*/otbn_top_sim.sv" -match "*'ImemStartAddr' generates 32 bits*"
 lint_off -rule WIDTH -file "*/otbn_top_sim.sv" -match "*'stack_wr_ptr_q' generates 4 bits*"
+lint_off -file "*/prim_subst_perm.sv"
+lint_off -file "*/prim_ram_1p_scr.sv"
```

Comment:
Whoah! These waivers need to be in the primitives (or, better yet, whatever they are waiving needs to be fixed). I know this is just a testbench, but there's no real reason not to do this right.

Reply:
+1

Reply:
I was taking the view that whilst the OTBN RTL is kept Verilator lint clean the rest of the OT RTL is not (though it is an aspiration to do so) so fixing prim lint didn't need to be part of this PR. I had specifically included the waiver for OTBN Verilator sim rather than in prim so the lint wouldn't disappear anywhere (in particular for the dashboard that tracks it all).

That said I can take a look at what's required to fix.

Reply:
Turned out to be very straight-forward, I should probably be less lazy about lint!